airmozilla/manage/static/manage/css/survey.css

```diff
@@ -0,0 +1,16 @@
+.survey-links{
+    display:inline-block;
+    margin-top:10px;
+    margin-bottom:10px;
+  }
+  td.question {
+    width: 80%;
+  }
+  textarea {
+    width: 100%;
+    height: 140px;
+  }
+  td.ordering {
+    vertical-align: middle;
+  }
+  
```

Comment:
This file needs some cleaning up. Consistent spacing, same indentation and the last line needs to be a `\n`.

E.g.
```python
>>> open('airmozilla/manage/static/manage/css/survey.css').read()[-1]
' '
>>> open('airmozilla/surveys/static/surveys/css/survey.css').read()[-1]
'\n'
```
That's why GitHub puts a little red stop and return icon here. 

What editor are you using? You should be able to configure it to automatically fix that on save.